scanpy/logging.py

```diff
@@ -145,11 +130,12 @@ def print_versions():
     Matplotlib and Seaborn are excluded from this.
     """
     from ._settings import settings
-    modules = ['scanpy'] + _DEPENDENCIES_NUMERICS
-    print(' '.join(
-        f'{mod}=={ver}'
-        for mod, ver in _versions_dependencies(modules)
-    ), file=settings.logfile)
+    stdout = sys.stdout
+    try:
+        sys.stdout = settings.logfile
```

Comment:
Should this be going to the log file? The function is called `print`, and I think it's up to the logging framework what do about `stdout`.

Reply:
True, those are called by the user manually so they don’t pollute stdout anyway even if scanpy is used as a library.